extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/SparkRangerAdminPlugin.scala

```diff
@@ -131,4 +140,39 @@ object SparkRangerAdminPlugin extends RangerBasePlugin("spark", "sparkSql")
       }
     }
   }
+
+  def getOrCreateRangerPlugin(catalog: Option[String] = None): RangerBasePlugin = synchronized {
```

Comment:
anywhere we set catalog?

Reply:
This will be done in the following PR in pull <https://github.com/apache/incubator-kyuubi/pull/3606> by adding `catalog` to `PrivilegeObject`. And then fill catalog info from extractors in another PR.

The initial plan and steps are in the umberalla issue here . https://github.com/apache/incubator-kyuubi/issues/3586